Tagesspiegel.js

```diff
@@ -82,8 +82,9 @@ function scrape(doc, url) {
 
 	newItem.title = ZU.xpathText(doc, "//meta[@property='og:title']/@content");
 	newItem.date = ZU.xpathText(doc, "//time[@itemprop='datePublished']/@datetime");
-	newItem.abstractNote = ZU.xpathText(doc, ".//p[@class='hcf-teaser']");
-	newItem.section = ZU.xpathText(doc, '//ul[contains(@class, "ts-main-nav-items")]/li[contains(@class, "ts-active-point")]/a');
+	newItem.abstractNote = ZU.xpathText(doc, '//meta[@name="description"]/@content');
+	// Note: doesn't work with subsections
```

Comment:
So this'll always grab the top-level section? That seems fine.

Reply:
Yes, I'll clarify the comment